WordPress/Classes/Models/Theme.h

```diff
@@ -22,6 +22,7 @@
 @property (nonatomic, retain) NSString *demoUrl;
 @property (nonatomic, retain) NSString *stylesheet;
 @property (nonatomic, retain) NSNumber *order;
+@property (nonatomic, retain) NSNumber *custom;
```

Comment:
I think you can just declare this as `BOOL` and skip the accessors. I see we already do that in `Blog.isMultiAuthor` for instance